galley/pkg/server/server.go

```diff
@@ -120,6 +120,13 @@ func newServer(a *Args, p patchTable) (*Server, error) {
 		}
 	}
 
+	defer func() {
```

Comment:
Can you move this to be closer to the server allocation up top?

Reply:
I considered before, due to there is unnecessary to execute `Close()` method when there are some errors thrown by `logConfigure`, `newMeshConfigCache`, etc. Anyway, let us move up to be closer to the server allocation which makes more sense.